reporting/vault-user-access-report.py

```diff
@@ -21,19 +23,32 @@ def getAllOwnerVaults():
         ["op", "vault", "list", "--group=Owners", "--format=json"], check=True, capture_output=True).stdout
     return vaultList
 
+
 def getSpecifiedVaults():
     with open(inputFilePath, "r", encoding="utf-8") as f:
         vaultList = []
         for id in f:
-            vaultList.append(json.loads(subprocess.run(["op", "vault", "get", id.rstrip(), "--format=json"], check=True, capture_output=True).stdout))
+            vaultList.append(json.loads(subprocess.run(["op", "vault", "get", id.rstrip(
+            ), "--format=json"], check=True, capture_output=True).stdout))
     return vaultList
 
 # get a list of users and their permissions for a vault
+
+
 def getVaultUserList(vaultID):
     vaultUserList = subprocess.run(
         ["op", "vault", "user", "list", vaultID, "--format=json"], check=True, capture_output=True).stdout
     return vaultUserList
 
+# get all groups in the account
```

Comment:
Why the extra spaces below here?

Reply:
VSCode's Python linter is.... opinionated in weird ways (see also your other formatting comment). I don't love some of those decisions either. Let me see if I can make some tweaks to how that behaves......

Reply:
just changed to a different auto formatter that's a bit less wonky. Minor improvement over autopep8, which is what was previously used.